### DISCUSSION TITLE
How to extract table without boarder but with lines inside the table

### DISCUSSION THREAD

**Discussion by MandyZ0** [jsvine/pdfplumber#793] [Ask for help with specific PDFs]
Hi all, I am trying the extract the data from this PDF: https://drive.google.com/file/d/1dieazEHC9MQGL9z_tEYIThkRXiWbvxDq/view?usp=sharing

How can I extract these tables when the vertial boarder is not right? Since I have bunch of PDFs to be processed, I might be not able to manually lable each corner for every PDF. What I'm thinking is like going through the position of each line and draw the boarder with min&max line position. However it might be interact with other lines such as the boarder of another table, or the top header line. 

Would you mind offering some ideas on how to solve this problem? Thanks so much for your kind help!
![Problematic_chart_example](https://user-images.githubusercontent.com/40691754/213579416-add0251c-8b57-45d4-baaa-c756e24c23c2.png)

**Comment by jsvine** [comment]
> What I'm thinking is like going through the position of each line and draw the boarder with min&max line position. However it might be interact with other lines such as the boarder of another table, or the top header line.

That's the approach I would suggest as well. To address your concerns, I'd suggest examining the detailed attributes of the page's `.line` and `.rect` objects. It appears that the top line and the bottom border of each table have different characteristics, which you could use to identify where each table begins and ends, possibly preventing the undesired interaction you mention.

**Reply by MandyZ0** [reply]
Oh I see, thanks so much for your suggestions! :D
